component/rules.jsonnet

```diff
@@ -104,6 +104,32 @@ local annotateRules = {
   },
 };
 
+local rulePatches = {
+  // upstream SystemMemoryExceedsReservation rule doesn't have `for` field
+  SystemMemoryExceedsReservation: {
+    'for': '15m',
+  },
+};
```

Comment:
Should we expose the option to provide rule patches in the hierarchy?

For example, we could add parameter `openshift4_monitoring.alerts.rulePatches` and use that instead of the local variable.

Reply:
Yes please.